doc/api/test.md

```diff
@@ -1222,6 +1222,7 @@ object, streaming a series of events representing the execution of the tests.
 ### Event: `'test:diagnostic'`
 
 * `data` {Object}
+  * `file` {string} The path of the test file.
```

Comment:
As the code is currently written, couldn't all of these be `undefined` too? For example, if `node:test` is used in the REPL.